packages/node-utilities/tsup.config.ts

```diff
@@ -0,0 +1,3 @@
+import { createTsupConfig } from '../../scripts/tsup.config';
+
+export default createTsupConfig({ target: 'es2019', format: ['cjs', 'esm'] });
```

Comment:
No need to target such low es, go for pandemic edition

Reply:
For added context, the reason /utilities does is because of Webpack 4 support which is sadly still widely used